packages/generate-cnpj/src/index.js

```diff
@@ -0,0 +1,20 @@
+import {
+  CNPJ_LENGTH,
+  FIRST_DIGIT_WEIGHTS,
+  SECOND_DIGIT_WEIGHTS,
+  generateRandomNumbers,
```

Comment:
Just to keep the consistency between our packages, I think that is better to move all function to `index.js` file

Reply:
the same thing for tests

Reply:
Changed in e29f8d0c082fc76f36ab586916a8c59e25fb998f